assets/sass/components/key-metrics/_googlesitekit-key-metrics-conversion-reporting-subtle-notification.scss

```diff
@@ -43,6 +43,7 @@
 	}
```

Comment:
SMall style adjustment purely to cover the edge case when both new events and lost events callouts are shown